app/controllers/api_controller.rb

```diff
@@ -66,4 +66,24 @@ def model
   def permitted_params
     raise "You haven't defined permitted params for #{self.class}"
   end
+
+  def per_page
```

Comment:
It's really odd that you had to copy this up to here. :thinking:  Do you have any idea why that was necessary?

Reply:
It didn't know the variable per_page. I was getting per_page undefined

Reply:
I see the issue now. The api controller doesn't inherit from the application controller.